### ISSUE TITLE
hugo complaining about i.license file

### ISSUE THREAD

**Issue by dancesWithCycles** [fsfe/reuse-website#38]
Hi folks,
Thank you very much for providing this repo. Please accept my appreciation! I did the following with this repository by now.

```
cd git/
git clone https://git.fsfe.org/reuse/website.git
cd website/site/
hugo server -D
```

If follows the response of the last hugo instruction here.

```
~/git/website/site$ hugo server -D
port 1313 already in use, attempting to use an available port
Total in 46 ms
Error: Error building site: "/home/user/git/website/site/data/organisations.json.license:1:1": unmarshal of format "" is not supported
```

It looks like hugo is not willing to swallow the *.license file. Had anyone else observed a behavior like this?

Cheers!

**Comment by mxmehl** [comment]
Hm, I cannot reproduce this with `hugo v0.88.1+extended linux/amd64`

What's your hugo version? In any case, perhaps it's better to ask this at hugo upstream? I don't think it makes semse that Hugo even considers this unused file in the data directory.

**Comment by dancesWithCycles** [comment]
Hi @mxmehl ,
Thank you very much for coming back promptly. It turns out my hugo version is not the youngest.

```
Hugo Static Site Generator v0.54.0/extended linux/amd64 BuildDate: 2019-06-04T19:16:36Z
```

This version is shipped with the latest packages in Debian 10.10.

I'll do as you recommended and discuss this matter with [hugo](https://github.com/gohugoio).

Cheers mate!
